changelog/_unreleased/2025-04-28-inclue-seourls-for-suggestsearch-and-search-entities.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Inclue seoUrls for suggestSearch and search entities
+issue: 489
+---
+# Core
+* Changed `Shopware\Core\Content\Seo\SalesChannel\StoreApiSeoResolver` class to include `seoUrls` for category in `suggestSearch` and `search` results.
```

Comment:
* Added `SeoUrlAwareExtensionInterface` to define the extension should be enriched with seoUrls
* Changed ... to allow enriching seoUrls for the search results extension